include/mtx/events/encrypted.hpp

```diff
@@ -143,9 +143,9 @@ struct KeyRequest
         std::string requesting_device_id;
 
         // //! The user that send this event.
-        // std::string sender;
+        std::string sender;
```

Comment:
The sender and EventType aren't required here, since they are populated from either DeviceEvent<type> (in the case of to-device messages), or in EncryptedEvent<type> (in the case of room messages).

Reply:
Okay I'll make the changes